publications-2020.bib

```diff
@@ -11,6 +11,87 @@ @book{Wi20_book
       url = "https://www.degruyter.com/view/title/523232"
 }
 
+@Article{HeiWi20,
+title = "pfm-cracks: A parallel-adaptive framework for phase-field fracture propagation",
+journal = "Software Impacts",
+volume = "6",
+pages = "100045",
+year = "2020",
+issn = "2665-9638",
+doi = "https://doi.org/10.1016/j.simpa.2020.100045",
+url = "http://www.sciencedirect.com/science/article/pii/S2665963820300361",
+author = "Timo Heister and Thomas Wick"
+}
+
+@Article{JoLaWi20,
+title = "Matrix-free multigrid solvers for phase-field fracture problems",
+journal = "Computer Methods in Applied Mechanics and Engineering",
+volume = "372",
+pages = "113431",
+year = "2020",
+issn = "0045-7825",
+doi = "https://doi.org/10.1016/j.cma.2020.113431",
+url = "http://www.sciencedirect.com/science/article/pii/S0045782520306162",
+author = "D. Jodlbauer and U. Langer and T. Wick"
+}
+
+@Article{JoLaWi20_parallel,
+author = {Jodlbauer, D. and Langer, U. and Wick, T.},
+title = {Parallel Matrix-Free Higher-Order Finite Element Solvers for Phase-Field Fracture Problems},
+journal = {Mathematical and Computational Applications},
+volume = {25},
+number = {3},
+pages = {40},
+year = {2020}
+}
```

Comment:
Do you have a `doi`/`url` for this one, too?

Reply:
doi = {https://doi.org/10.3390/mca25030040}

Reply:
I mean would you please add the `doi` to your commit :-)

Reply:
Done. Added doi into commit.